Update docs (#314)
cesium-ml/cesium: doc/install.md

# Installation

## From binaries:

> - Wheels for Mac and Linux can be installed via `pip install cesium`

Reviewer: ```suggestion
- `pip install cesium`
```

> - We do not build binary wheels for Windows. To install on Windows, follow the instructions below for installation from source.

Reviewer: I did publish 64-bit Windows wheels for the last release. I cannot confirm whether they work or not.

Either way, I think this section should be removed.

The `cesium` library runs on Python 3.

## From source:
1. Install [Cython](http://cython.readthedocs.io/en/latest/src/quickstart/install.html)
2. Clone the repository: `git clone https://github.com/cesium-ml/cesium.git`
3. `cd cesium && pip install -e .`

> Note that cesium requires a C99 compiler, which in particular excludes MSVC. On Windows, a different compiler like MinGW has to be used. Please refer to the [instructions for installing Cython & MinGW on Windows](https://cython.readthedocs.io/en/latest/src/tutorial/appendix.html#appendix-installing-mingw-on-windows).

Reviewer: ```suggestion
Note that cesium requires a C99 compiler, which in particular excludes MSVC. On Windows, we recommend clang.
```
